### DISCUSSION TITLE
Is it okay to build a service that reads twitter timeline?

### DISCUSSION THREAD

**Discussion by mariotaku** [mastodon/mastodon#20835] [Q&A] [answered]
**Disclaimer**: This is **NOT** a service performing crossposting from/to twitter, and **NOT** a service that creates fake bot users from twitter.

Some friends that doesn't intend to move to Mastodon, might be one's biggest reason of continue using Twitter.

Some client apps can solve such issue, like Twidere (which has combined timeline). How about a service doing that?

I'm thinking to create something connecting 2 identities (Mastodon, Twitter), and let the user search the Twitter user they have followed within Mastodon, seeing their timeline and additionally, interacting with them if possible.

If those contents from twitter can't be reposted, but only available to the Twitter user, I think it will not create spam.

What do you think? Does it against community guidelines of most Mastodon instances?

**Comment by ineffyble** [comment, marked as answer]
Such services already exist, but they've caused a fair few issues. Here's a blog post explaining why one of the prominent ones, BirdsiteLIVE, was turned off a week ago: https://write.as/nicolas-constant/closing-the-official-bsl-instance

**Reply by mariotaku** [reply]
Thanks for your input. That kind of service is indeed what I'm looking for. The conclusion would be better avoid such usage (or at least run on private instances only).

**Comment by markcellus** [comment]
Have you considered following the twitter users using RSS feeds?
